libr/core/rvc.c

```diff
@@ -227,9 +228,26 @@ static RList *get_commits(const char *rp, const size_t max_num) {
 	return ret;
 }
 
-static bool update_blobs(RList *blobs, const RList *nh) {
+
+static bool in_rvc_ignore(const char *rp, const char *rpf) {
+	char *ignorep = r_str_newf ("%s" R_SYS_DIR IGNORE_NAME, rp);
```

Comment:
Didnt u wanted to use the rpath api?

Reply:
It didn't work for some reason appending weird strings to the end

Reply:
Last arg must be NULL